packages/mui-base/src/useList/listReducer.ts

```diff
@@ -103,6 +103,16 @@ export function moveHighlight<ItemValue>(
       wrapAround = false;
       break;
 
+    case 'reset-to-end':
```

Comment:
`"end"` should be sufficient in this case. I don't think there's a need for an additional option here.